internal/versions/versions.go

```diff
@@ -85,11 +101,11 @@ const (
 
 	// currently supported versions.
 	//nolint:revive
-	V1_24 ValidK8sVersion = "1.24"
+	V1_24 ValidK8sVersion = "v1.24.9"
 	//nolint:revive
-	V1_25 ValidK8sVersion = "1.25"
+	V1_25 ValidK8sVersion = "v1.25.6"
 	//nolint:revive
-	V1_26 ValidK8sVersion = "1.26"
+	V1_26 ValidK8sVersion = "v1.26.1"
 
```

Comment:
Checked our code again, if I understand it correctly the `V1_X ValidK8sVersion` variables are only used to keep track of the minor Kubernetes version supported by Constellation (We don't support choosing patch versions).

I think we should keep them as only Major.Minor, not sure about the `v` prefix.

Reply:
The reason for switching to exposing the patch version is that we might want to offer a patch-version upgrade to the user. I think its cleaner to show users that the version will change from e.g. `1.25.6` to `1.25.7` instead of doing this implicitly. Users will still not be able to chose arbitrary patch versions, but it will be obvious which one is running/upgraded.